packages/client/tests/functional/esbuild-transformer.js

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs')
+
+const esbuild = require('esbuild')
+const getCacheKeyFunction = require('@jest/create-cache-key-function').default
+
+const cacheKeyFunction = getCacheKeyFunction([], ['v1'])
+
+function needsTranspilation(contents, filename) {
+  if (filename.endsWith('.ts') === true) {
+    return true
+  }
+
+  if (contents.includes('require(') === true) {
+    return false
+  }
+
+  if (contents.includes('module.exports') === true) {
+    return false
+  }
+
+  if (contents.includes('exports.') === true) {
+    return false
+  }
+
+  return false
```

Comment:
Since the default is `return false`, the conditions that also return false are not really needed?

Reply:
I forgot a condition